### PR TITLE
Proofread FAQ

### PR DESCRIPTION
**Pull request by kdaily**
Addresses #1

### CODE REVIEW

File: articles/faq.md [hunk 1]
```diff
@@ -9,92 +9,86 @@ layout: article
...
 
 #### Is there someone that can help me use Synapse in my collaboration?
- Our [Discussion Forum](https://www.synapse.org/#!SynapseForum:default) is a great place to reach out to the broader Synapse community to find others that may be interested in a collaboration. You can also reach us directly via email at SynapseInfo@sagebase.org
+ Our [Discussion Forum](https://www.synapse.org/#!SynapseForum:default) is a great place to reach out to the broader Synapse community to find others that may be interested in a collaboration. You can also reach us directly via email at [SynapseInfo@sagebase.org](mailto:SynapseInfo@sagebase.org).
 
-
-#### What are the Synapse Terms  of Use?
-In order to register on Synapse, you must review and agree to the terms of the Synapse Awareness and Ethics Pledge. These terms are described in more detail in the Synapse [Terms and Conditions of Use document](https://s3.amazonaws.com/static.synapse.org/governance/SageBionetworksSynapseTermsandConditionsofUse.pdf?v=4). By using Synapse you must further consent to the Synapse [Governance policies](https://www.synapse.org/#!Help:Governance)
+#### What are the Synapse Terms of Use?
```
Comment (larssono) [inline articles/faq.md]: We should migrate Governance over as well

File: articles/faq.md [hunk 2]
```diff
@@ -9,92 +9,86 @@ layout: article
...
+Yes, Synapse is published under [The MIT Licence](https://github.com/Sage-Bionetworks/file-proxy/blob/develop/LICENSE) The source code is available on the [Sage Bionetworks GitHub page](https://github.com/Sage-Bionetworks/). Synapse is also offered free of charge as a hosted Software as a Service (SaaS).
 
-#### I am a developer - is there  Application Programmatic Interface (API) documentation?
-Yes, Synapse is built on top of a RESTful service that is automatically (documented)(TODO update link: http://hud.rel.rest.doc.sagebase.org.s3-website-us-east-1.amazonaws.com). In addition, we have purpose built APIs for (Python)[TODO: Link], (R )[TODO: R documentation), (Java)[TODO java docs) and a (command line)[TODO: command line docs] interface.
+#### I am a developer - is there API documentation?
+Yes, Synapse is built on top of a RESTful service that is automatically [documented](http://rest.synapse.org). In addition, we have purpose built APIs for [Python](TODO: Link), [R](TODO: R documentation), [Java](TODO java docs) and a [command line](TODO: command line docs) interface.
```
Comment (larssono) [inline articles/faq.md]: Update link to docs.synapse.org/rest

File: articles/faq.md [hunk 3]
```diff
@@ -9,92 +9,86 @@ layout: article
...
+Synapse was developed with the philosophy to encourage collaboration across institutional boundaries and is therefore provided as “Software As A Service” with a single instance used by all users. This makes it easy both to discover new content and share with new collaborators. We do support private project spaces where content sharing is controlled by the individual user. In addition, Synapse has the ability to reference resources that are stored elsewhere. This allows Synapse to store metadata about the content such as annotations, descriptive wiki pages and provenance but not the actual data. Currently Synapse has specific support for files stored at URLs, on SFTP servers, on AWS S3 and arbitrary file servers (TODO: add links to each one of these on how to setup).
```
Comment (larssono) [inline articles/faq.md]: Link is available /articles/custom_storage_location.html

File: articles/faq.md [hunk 4]
```diff
@@ -9,92 +9,86 @@ layout: article
...
 
 #### How do I get started?
-Whether you are looking to access content or use Synapse to track your own work start by [registering](https://www.synapse.org/#!RegisterAccount:0) for a Synapse user account. Before uploading files, you will need to take a short [certification quiz](https://www.synapse.org/#!Quiz:Certification) that demonstrates your understanding of the ethical considerations in sharing dataand the manner in which data is managed and shared in Synapse. See the Getting Started guide (TODO: link) for tips on how to access data, create your own project and share content with others.
+Whether you are looking to access content or use Synapse to track your own work, start by [registering](https://www.synapse.org/#!RegisterAccount:0) for a Synapse user account. Before uploading files, you will need to take a short [certification quiz](https://www.synapse.org/#!Quiz:Certification) that demonstrates your understanding of the ethical considerations in sharing data and the manner in which data is managed and shared in Synapse. See the Getting Started guide (TODO: link) for tips on how to access data, create your own project and share content with others.
```
Comment (larssono) [inline articles/faq.md]: Add link to /articles/getting_started.html

File: articles/faq.md [hunk 5]
```diff
@@ -9,92 +9,86 @@ layout: article
...
 
 ## Accessing Content
 
 #### My colleague put some content in Synapse.  How do I find it?
-This will depend if the content is public or private. If private, you will need to make sure your colleague has shared this content with you. Shared content is visible from your “Dashboard page” under the tab “Shared directly with me”. If you favorite the content it will appear under your list of favorites visible on any Synapse page.
+This will depend if the content is public or private. If private, you will need to make sure your colleague has shared this content with you. Shared content is visible from your “Dashboard page” under the tab “Shared directly with me”. If you favorite the content (using the yellow star) it will appear under your list of favorites visible from the top of any Synapse page.
```
Comment (larssono) [inline articles/faq.md]: Provide link #!Profile:v/...

File: articles/faq.md [hunk 6]
```diff
@@ -9,92 +9,86 @@ layout: article
...
 
-All public data is queryable (for more information see [help on querying](TODO link to query docs) or from the “Search” box in the top right corner of any Synapse page.
+All public data is queryable. For more information see [help on querying](TODO link to query docs) or from the “Search” box in the top right corner of any Synapse page.
```
Comment (larssono) [inline articles/faq.md]: Add link to /articles/annotation_and_query.html

File: articles/faq.md [hunk 7]
```diff
@@ -9,92 +9,86 @@ layout: article
...
 
 ## Adding Content
 
 #### Why do I have to be a certified user to upload content?
-User certification ensures that you understand your responsibilities for sharing data through Synapse, especially data derived from human participants.  These responsibilities include making sure that data derived from human participants is de-identified according to HIPAA standards, and that all applicable privacy laws and regulations are observed.  See the Data Use Procedure for moreinformation.
+User certification ensures that you understand your responsibilities for sharing data through Synapse, especially data derived from human participants. These responsibilities include making sure that data derived from human participants is de-identified according to [HIPAA standards](http://www.hhs.gov/hipaa/for-professionals/privacy/special-topics/de-identification/index.html), and that all applicable privacy laws and regulations are observed.  See the Data Use Procedure for more information.
```
Comment (larssono) [inline articles/faq.md]: HIPAA US centric.  We should probably be more broad

File: articles/faq.md [hunk 8]
```diff
@@ -9,92 +9,86 @@ layout: article
...
 
+#### Is everything I share on Synapse public?
+No. You can share your content which whomever you chose. By default, Projects and their content are visible only to the user who created it. You have the ability to grant other Synapse users, Synapse Teams, or the public access to your Project content.
 
 #### Can I store sensitive information about human subjects in Synapse?
-Yes.  Synapse has an IRB-approved data governance procedure that provides mechanisms to share sensitive data in a controlled manner.  You can learn more by reading our Data Use Procedure documentation.  For further inquires or help in understanding how to apply these procedures to your data, please contact the Synapse Access and Compliance Team: act@sagebase.org
+Yes. Synapse has an IRB-approved data governance procedure that provides mechanisms to share sensitive data in a controlled manner. You can learn more by reading our Data Use Procedure documentation. For further inquires or help in understanding how to apply these procedures to your data, please contact the Synapse Access and Compliance Team at [act@sagebase.org](mailto:act@sagebase.org).
```
Comment (larssono) [inline articles/faq.md]: Add link to Governance help

File: articles/faq.md [hunk 9]
```diff
@@ -9,92 +9,86 @@ layout: article
...
 
 #### How do I get certified?
-To become a certified user, you will need to pass a brief quiz  that may accessedhere. 
-
-#### Is everything I share on Synapse Public?
-No. You can share your content which whomever you chose. By default, Projects and  project content are visible only to the user who created it.You have the ability to grant other Synapse users, Synapse Teams, or the public access to your Project content.
+To become a certified user, you will need to pass a brief quiz that may accessed here.
 
+#### Is everything I share on Synapse public?
+No. You can share your content which whomever you chose. By default, Projects and their content are visible only to the user who created it. You have the ability to grant other Synapse users, Synapse Teams, or the public access to your Project content.
```
Comment (larssono) [inline articles/faq.md]: with whomever ...

File: articles/faq.md [hunk 10]
```diff
@@ -9,92 +9,86 @@ layout: article
...
 
-#### How do I know the content I put in Synapse will be secure (what security measures does Synapse have?)
-Synapse stores content in Amazon Web Services, which provides a layer of security measures designed and implemented by Amazon (TODO: link to Amazon security documentation). While Synapse is an open access site, each user has control over who may access their content . Only users whoyou share content with see it.You can set different sharing settings for individual files anf folders in your project.
+#### How do I know the content I put in Synapse will be secure? What security measures does Synapse have?
+Synapse stores content in Amazon Web Services, which provides a layer of security measures designed and implemented by Amazon (TODO: link to Amazon security documentation). While Synapse is an open access site, each user has control over who may access their content. Only users with whom you share content will see it. You can set different sharing settings for all individual Files and Folders in your Project.
```
Comment (larssono) [inline articles/faq.md]: Update link

File: articles/faq.md [hunk 11]
```diff
@@ -9,92 +9,86 @@ layout: article
...
-By default, Synapse stores files in Amazon Simple Storage Services (S3). However it is possible to set up Synapse to store files in different locations such as your local SFTP server. For files stored outside of S3, Synapse can be used to organize, manage, and access files through the use of the Synapse annotations system to store file-specific metadata.  (link to an example)
-
-#### I have my research results - How can Synapse help me share them ?
-Synapse makes it easy to share files of any sort, with whomever you choose whether a small group of collaborators or the general public. You may share raw data, summarized data, analysis results, or anything in between. We recommend using “Getting started with Synapse guide” for more details on how to upload files and content and manage access permissions.
+By default, Synapse stores files in Amazon Simple Storage Services (S3). However it is possible to set up Synapse to store files in different locations such as your local SFTP server. For files stored outside of S3, Synapse can be used to organize, manage, and access files through the use of Synapse annotations to store file-specific metadata.  (link to an example)
```
Comment (larssono) [inline articles/faq.md]: Add actual link

**Comment by kdaily** [comment]
@larssono  - fixed everything except a link to some example. Does this already exist?

**Pull request merged**